diff --git a/node_modules/histoire/dist/node/collect/index.js b/node_modules/histoire/dist/node/collect/index.js
index 8ffd507..37be43e 100644
--- a/node_modules/histoire/dist/node/collect/index.js
+++ b/node_modules/histoire/dist/node/collect/index.js
@@ -22,7 +22,7 @@ export function useCollectStories(options, ctx) {
                 /vite\w*\/dist\/client\/(client|env).mjs/,
                 ...ctx.config.viteNodeInlineDeps ?? [],
             ],
-            fallbackCJS: true,
+            fallbackCJS: false,
         },
         transformMode: ctx.config.viteNodeTransformMode,
     });
