diff --git a/dist/Select/SelectItem.cjs b/dist/Select/SelectItem.cjs
index 585652ced03ead8a2198fb2062bb9e6dc077df8e..0de0b74277161035e8d85c0fd320e084e791c4f5 100644
--- a/dist/Select/SelectItem.cjs
+++ b/dist/Select/SelectItem.cjs
@@ -60,7 +60,7 @@ const _sfc_main = /* @__PURE__ */ vue.defineComponent({
       if (disabled.value) {
         contentContext.onItemLeave?.();
       } else {
-        event.currentTarget.focus({ preventScroll: true });
+        event.currentTarget?.focus({ preventScroll: true });
       }
     }
     async function handlePointerLeave(event) {
diff --git a/dist/Select/SelectItem.js b/dist/Select/SelectItem.js
index 8d538931e6640e3cac01fc031962ebab513a4e44..9b10559b4f29b15f0ce3181667c348b19c91b7b6 100644
--- a/dist/Select/SelectItem.js
+++ b/dist/Select/SelectItem.js
@@ -58,7 +58,7 @@ const _sfc_main = /* @__PURE__ */ defineComponent({
       if (disabled.value) {
         contentContext.onItemLeave?.();
       } else {
-        event.currentTarget.focus({ preventScroll: true });
+        event.currentTarget?.focus({ preventScroll: true });
       }
     }
     async function handlePointerLeave(event) {
